deploy/charts/cert-manager/values.yaml

```diff
@@ -28,43 +30,65 @@ global:
     # leadership renewal until attempting to acquire leadership of a led but
     # unrenewed leader slot. This is effectively the maximum duration that a
     # leader can be stopped before it is replaced by another candidate.
+    # Example:
     # leaseDuration: 60s
+    leaseDuration:
 
     # The interval between attempts by the acting master to renew a leadership
     # slot before it stops leading. This must be less than or equal to the
     # lease duration.
+    # Example:
     # renewDeadline: 40s
+    renewDeadline:
 
     # The duration the clients should wait between attempting acquisition and
     # renewal of a leadership.
+    # Example:
     # retryPeriod: 15s
+    retryPeriod:
 
+# If true, CRD resources will be installed as part of the Helm chart.
+# If enabled, when uninstalling CRD resources will be deleted causing all
+# installed custom resources to be DELETED.
 installCRDs: false
 
+# Number of cert-manager replicas
 replicaCount: 1
 
+# Example:
+# strategy:
+#   type: RollingUpdate
+#   rollingUpdate:
+#     maxSurge: 0
+#     maxUnavailable: 1
```

Comment:
I started wondering if it'd be niced to parse the comments as Markdown and then the example could be in a fenced code block labelled:
 ` ```example `
```
strategy:
   type: RollingUpdate
   rollingUpdate:
     maxSurge: 0
     maxUnavailable: 1
```
` ``` `

Reply:
The main problem is that a markdown table can't easily contain a code snippet. You can't just do
\`\`\`yaml
strategy:
   type: RollingUpdate
\`\`\`
inside a table.
Instead, it has to be formatted as <pre lang=\"yaml\">strategy:\&NewLine;   type: RollingUpdate\<\/pre>